src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String inputCommand;
+        printGreeting();
+
+        // Command handler
+        do {
+            Scanner in = new Scanner(System.in);
+            inputCommand = in.nextLine();
+
+            String taskObj = new Scanner(inputCommand).next();
+
+            switch (taskObj){
+                case "list":
+                    listAllTasks();
+                    break;
+
+                case "bye":
+                    printGoodbye();
+                    break;
+
+                case "done":
+                    String[] commandWords = inputCommand.split(" ");
+                    int taskNumber = Integer.parseInt(commandWords[1])-1;
+                    markTaskAsDone(taskNumber);
+                    break;
+
+                case "todo":
+                    String taskDetails = inputCommand.substring(4);
+                    addTask(new ToDo(taskDetails));
+                    break;
+
+                case "deadline":
+                    taskDetails = inputCommand.substring(8);
+                    String[] detailsWords = taskDetails.split("/by ");
+                    addTask(new Deadline(detailsWords[0], detailsWords[1]));
+                    break;
+
+                case "event":
+                    taskDetails = inputCommand.substring(5);
+                    detailsWords = taskDetails.split("/at ");
+                    addTask(new Event(detailsWords[0], detailsWords[1]));
+                    break;
+            }
+        } while (!inputCommand.equals("bye"));
+    }
+
+
+    // Prints greeting and logo
+    static void printGreeting(){
+        String logo =     "  ____  ____   _      _       ____  ___ ___   ____ \n"
+                        + " /    ||    \\ | |    | |     /    ||   |   | /    |\n"
+                        + "|  o  ||  _  || |    | |    |  o  || _   _ ||  o  |\n"
+                        + "|     ||  |  || |___ | |___ |     ||  \\_/  ||     |\n"
+                        + "|  _  ||  |  ||     ||     ||  _  ||   |   ||  _  |\n"
+                        + "|  |  ||  |  ||     ||     ||  |  ||   |   ||  |  |\n"
+                        + "|__|__||__|__||_____||_____||__|__||___|___||__|__|";
+        System.out.println(LINE_HEADER + "\t\t\t\t\tHello from\n" + logo);
+        System.out.println(LINE_HEADER+ "\tHello! I'm your friendly neighbourhood Llama.\n\tWhat can I do for you?" + LINE_HEADER);
+    }
+
+    // Add task to taskList
+    static void addTask(Task task){
+        taskList[taskCount] = task;
+        taskCount++;
+        System.out.println(LINE_HEADER + "\tAdded: " + task + "\nNow you have " + taskCount + " task(s) in your list!" + LINE_HEADER);
+    }
+
+    // Marks task in taskList as done
+    static void markTaskAsDone(int taskNumber){
```

Comment:
methods are generally abstracted, which will improve readability further down the road. good job